docs/designers-developers/developers/tutorials/javascript/js-build-setup.md

```diff
@@ -119,13 +119,13 @@ To configure npm to run a script, you use the scripts section in `package.json`
 
 You can then run the build using: `npm run build`.
 
-After the build finishes, you will see the built file created at `build/index.js`. Enqueue this file in the admin screen as you would any JavaScript in WordPress, see [Step 2 in this tutorial](loading-javascript.md), and the block will load in the editor.
+After the build finishes, you will see the built file created at `build/index.js`. Enqueue this file in the admin screen as you would any JavaScript in WordPress, see [Step 2 in this tutorial](extending-the-block-editor/), and the block will load in the editor.
```

Comment:
Hi @dhurlburtusa thank you for your contribution 👍  After checking both links, It seems the previous link was correct, it was linking to step two linked on https://github.com/WordPress/gutenberg//blob/0ad0414e7a469254ef1ad105fdcf14248a986686/docs/designers-developers/developers/tutorials/javascript/readme.md#L15. And the previous text refers the mechanism to enqueue which is documented on loading-javascript.md.

Reply:
Thanks Jorge for confirming the correct destination. I updated the PR to point to the previous destination but with the correct URL.

Reply:
Thanks @jorgefilipecosta for confirming the correct destination. I wasn't sure what step was being referred to when it said "Step 2 in this tutorial".

I am confused on the best way to fix this link. Should it be changed to point to `readme.md` since the link text ("Step 2 in this tutorial") refers to the tutorial itself? Or should we link directly to step 2 which would be `loading-javascript.md`. In that case, I think the link text should be updated.

There is also some confusion as to whether I should be leaving the extension on (`.md`). In GitHub, it seems the extension is necessary. Otherwise navigating within GitHub is broken. But when I go to the hosted website version, none of the extensions exist. I guess some build process removes them. And therein lies the problem. Here is the file structure:

```
...
js-build-setup.md
loading-javascript.md
...
readme.md
...
```
The problem is that in GitHub, `js-build-setup.md` and `loading-javascript.md` are sibling _**files**_. So linking with `loading-javascript.md` works correctly. But on the web side of things, they are sibling _**paths**_. So linking to `../loading-javascript/` is required.

I've seen in https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/tutorials/javascript/loading-javascript.md that some links take you out of GitHub and to the website (developer.wordpress.org). But in other places such as https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/tutorials/javascript/readme.md uses relative references to the markdown file.

So, I am not sure the recommended way to fix this link.

Reply:
Hi @dhurlburtusa, 

> I am confused on the best way to fix this link. Should it be changed to point to readme.md since the link text ("Step 2 in this tutorial") refers to the tutorial itself? Or should we link directly to step 2 which would be loading-javascript.md. In that case, I think the link text should be updated.

I guess we should link to loading javascript and that it makes sense to update the link text.


> The problem is that in GitHub, js-build-setup.md and loading-javascript.md are sibling files. So linking with loading-javascript.md works correctly. But on the web side of things, they are sibling paths. So linking to ../loading-javascript/ is required.

Interesting point. Would link to something like `../../javascript/loading-javascript.md` work on GitHub and hosted website version?